pkg/gardenlet/controller/seed/seed/reconciler_reconcile.go

```diff
@@ -886,6 +880,16 @@ func (r *Reconciler) runReconcileSeedFlow(
 		return err
 	}
 
+	// TODO(rfranzke): Delete this in a future version.
+	{
+		if err := kutil.DeleteObjects(ctx, seedClient,
+			&resourcesv1alpha1.ManagedResource{ObjectMeta: metav1.ObjectMeta{Name: "gardener-seed-admission-controller", Namespace: r.GardenNamespace}},
+			&corev1.Secret{ObjectMeta: metav1.ObjectMeta{Name: "managedresource-gardener-seed-admission-controller", Namespace: r.GardenNamespace}},
+		); err != nil {
+			return err
+		}
+	}
```

Comment:
Any reason for extra curly braces here?

Reply:
For such temporary migration code, I like this approach because it simplifies identifying the to-be-dropped code (especially when it has multiple lines which is not the case here, but still). :)